support/k8s/k8s-workload-registrar/controller.go

```diff
@@ -114,8 +115,18 @@ func (c *Controller) createPodEntry(ctx context.Context, pod *corev1.Pod) error
 		return nil
 	}
 
-	// the controller has not been configured with a pod label. create an entry
-	// based on the service account.
+	if c.c.PodAnnotation != "" {
```

Comment:
Hmm, if both `PodLabel` and `PodAnnotation` are set, then `PodAnnotation` will effectively be ignored. For now, I think it's best if we make the configurables mutually exclusive and change `LoadConfig()` to fail if both are being set. Otherwise, we have to come up with some sort of preference.

Reply:
@azdagron I fixed it with https://github.com/spiffe/spire/pull/1047/commits/00b897fe6a42172cef9225fcd2c5a18910243b74.